hcl/hcl.go

```diff
@@ -1117,8 +1135,6 @@ func parseGitConfig(git *GitConfig, gitBlock *ast.MergedBlock) error {
 
 			git.DefaultBranch = value.AsString()
 		case "default_remote":
-			logger.Trace().Msg("Attribute name was 'default_remote'.")
-
 			if value.Type() != cty.String {
 				errs.Append(errors.E(attr.NameRange,
```

Comment:
```suggestion
				errs.Append(errors.E(attr.Expr.Range(),
```
not related to your change but just noticed this is wrong here.

Reply:
Extracted on a function trying to help being consistent about this.